pract_02.py

```diff
@@ -0,0 +1,29 @@
+def bigger_price(n, products):
+    products.sort(key=lambda dic: dic['price'], reverse=True)
+    print([products[i] for i in range(n)])
+
+bigger_price(2, [
+{"name": "bread", "price": 100},
+{"name": "wine", "price": 138},
+{"name": "meat", "price": 15},
+{"name": "water", "price": 1}
+])
+
+#================================================================
+
+import requests
+import json
+
+def list_of_names(imported):
+    return [elem['name'] for elem in imported]
+
+def names_weapons(imported):
+    return {elem['name']: elem['weapon'] for elem in imported if elem['weapon'] is not ''}
+
+data = json.loads(requests.get("https://lego-super-heroes.herokuapp.com/").text)
```

Comment:
Good, but I'm assuming to have some `def get_all_heroes_content()` function and put this code there.